src/gmail.js

```diff
@@ -3984,6 +4036,19 @@ var Gmail = function(localJQuery) {
                 show_cc: "span.aB.gQ.pE",
                 show_bcc: "span.aB.gQ.pB"
             };
+
+            // Post "peoplekit"
+            if (this.$el.find("textarea[name=to]").length === 0 && ["to", "cc", "bcc"].includes(lookup)) {
+                const configPeoplekit = {
+                    to:"div[name=to] input",
+                    cc:"div[name=cc] input",
+                    bcc:"div[name=bcc] input"
+                };
```

Comment:
I think it would be cleaner if we instead of having our own "full" branch for the entire code-sequence (lookup + retrieve), just used it to patch the `config` variable we already have.

That way we have 1 map, 1 patch and 1 general code-path for all lookups.